modules/python/pylib/Makefile.am

```diff
@@ -66,13 +66,8 @@ clean-pylib:
 
 python-checks: python-unit python-pep8 python-pylint
 
-if NOSETESTS
 python-unit:
-	@NOSETESTS@ $(PYLIB_SRCDIR)/syslogng
-else
-python-unit:
-	$(error "missing nosetests")
```

Comment:
Is it possible to keep an error message about a missing dependency here ?

Reply:
Of course it was in my mind. 
I have just updated the code.